source/transactions/tests/README.rst

```diff
@@ -98,7 +98,14 @@ For each YAML file, for each element in ``tests``:
    - Enter a "try" block or your programming language's closest equivalent.
    - If ``name`` is "startTransaction", "commitTransaction", or
      "abortTransaction", call the named method on ``session0`` or
-     ``session1``, depending on the "session" argument.
+     ``session1``, depending on the "session" argument. 
+   - If ``name`` is "runCommand", call the runCommand method on the database
+     specified in the test. Pass the argument named "command" to the runCommand 
+     method. Pass ``session0`` or ``session1`` to the runCommand method, depending 
+     on which session's name is in the arguments list. If ``arguments`` 
+     contains no "session", pass no explicit session to the method. If ``arguments`` 
+     includes "readPreference", also pass the read preference to the runCommand 
+     method.    
```

Comment:
This text explaining how to resolve the ambiguity of readPreference would not be necessary if the ambiguity didn't exist in the first place.

Shane opened: https://jira.mongodb.org/browse/SPEC-1097